client/blocks/feedback/edit.js

```diff
@@ -248,12 +249,15 @@ const EditFeedbackBlock = ( props ) => {
 			null !== attributes.closedAfterDateTime &&
 			new Date().toISOString() > attributes.closedAfterDateTime );
 
-	const hideBranding = get( accountInfo, 'capabilities' ).includes(
+	const hideBranding = get( accountInfo, 'capabilities', [] ).includes(
```

Comment:
Nit, I think this would be a little more concise:

```
const hideBranding = includes( accountInfo.capabilities || [], 'hide-branding' );
```

Reply:
Isn't one of `lodash.get`'s missions to ensure the same output even if `accountInfo` is `undefined` or `null`? What I'm trying to say is the use of `get` is so we don't have to check `accountInfo && accountInfo.capabilities`, we just rely on `get` to try that and provide the default if it fails. Syntax does look more concise though, just making sure to not throw safe mechanisms out the window in favor of neat syntax.

That said, yes, I try to make sure `accountInfo` always has some value, but better safe than uncontrolled errors :)